controls/pcidss_4_ocp4.yml

```diff
@@ -2498,36 +2485,41 @@ controls:
         Visitor identification or badges cannot be reused after expiration.
       levels:
         - base
-      status: pending
+      status: not applicable
 
     - id: 9.3.4
       title: A visitor log is used to maintain a physical record of visitor activity within the
         facility and within sensitive areas.
       levels:
         - base
-      status: pending
+      status: not applicable
 
   - id: '9.4'
     title: Media with cardholder data is securely stored, accessed, distributed, and destroyed.
     levels:
       - base
-    status: pending
+    status: not applicable
     controls:
     - id: 9.4.1
       title: All media with cardholder data is physically secured.
       description: |-
         Media with cardholder data cannot be accessed by unauthorized personnel.
       levels:
         - base
-      status: pending
+      status: not applicable
```

Comment:
I was wondering whether this requirement isn't actually `manual`, instead of `not applicable`

Reply:
I could see several of these controls being `manual` if you consider the physical perspective (e.g., you need to build fences around your cluster to meet requirement 9.2).

Marking them as manual seems to align with what RHEL did. Either way, I don't think these statuses will show up in the actual operator since they're not backed by any rules.